compute/Dockerfile.compute-node

```diff
@@ -869,6 +869,28 @@ RUN case "${PG_VERSION}" in "v17") \
     cargo pgrx install --release && \
     echo "trusted = true" >> /usr/local/pgsql/share/extension/ulid.control
 
+#########################################################################################
+#
+# Layer "pg-session-jwt-build"
+# Compile "pg_session_jwt" extension
+#
+#########################################################################################
+
+FROM rust-extensions-build AS pg-session-jwt-build
+ARG PG_VERSION
+
+RUN case "${PG_VERSION}" in "v17") \
+    echo "pg_session_jwt does not yet have a release that supports pg17" && exit 0;; \
+    esac && \
+    wget https://github.com/neondatabase/pg_session_jwt/archive/793e7268117d81f5f4268b7a07d17c9b61d4119e.tar.gz -O pg_session_jwt.tar.gz && \
+    echo "48277a7d4c53a6e52f4af4495f53250374c30dc06c1a20690a99d5b899c55e69 pg_session_jwt.tar.gz" | sha256sum --check && \
+    mkdir pg_session_jwt-src && cd pg_session_jwt-src && tar xzf ../pg_session_jwt.tar.gz --strip-components=1 -C . && \
+    sed -i 's/pgrx = "=0.11.3"/pgrx = { version = "=0.11.3", features = [ "unsafe-postgres" ] }/g' Cargo.toml && \
```

Comment:
Seems like we can be a little nicer to ourselves, and do a capture on the version numbers to reuse on the right hand side of the sed command.

Reply:
I think in some of these cases we'll want to change the pgrx version.

Also, I think one could argue that this is a bit harder to read:

```
    # Capture the pgrx version inbetween quotes and replicate it with \1 on the replacement
    sed -i 's/pgrx = "\([^"]*\)"/pgrx = { version = "\1", features = [ "unsafe-postgres" ] }/g' Cargo.toml && \
```

But I'm sending another PR very soon that will update all pgrx extensions to 0.12.x to support pg17, maybe this can be discussed more holistically there.